src/tree.c

```diff
@@ -334,7 +334,7 @@ static const imask_t has_map[T_LAST_TREE_KIND] = {
    (I_IDENT | I_ASSOCS | I_STMTS | I_DECLS),
 
    // T_PSL
-   (I_IDENT | I_FOREIGN),
+   (I_IDENT | I_PORTS | I_FOREIGN),
```

Comment:
This is not actually used, or necessary I think. The port list should go in the `psl_node_t`.